lib/hal/led.c

```diff
@@ -0,0 +1,27 @@
+#include <hal/led.h>
```

Comment:
As most of the existing HAL code is OpenXDK licensed, it might be a good idea to add a license header here (with a more suitable license).

*To avoid any license confusion.*